### PR TITLE
Minor fixes about bundle overridding

### PR DESCRIPTION
**Pull request by javiereguiluz**
This finishes #6985.

### CODE REVIEW

**Comment by Jean85** [comment]
LGTM :+1: 
Status: reviewed

File: bundles/override.rst [hunk 1]
```diff
@@ -72,17 +72,8 @@ associations. Learn more about this feature and its limitations in
...
-work if the type is referred to by its alias rather than being instantiated,
-e.g.::
-
-    $builder->add('name', 'custom_type');
-
-rather than::
-
-    $builder->add('name', new CustomType());
+Existing form types can be modified defining "form type extensions".
+For more information, see :doc:`/form/create_form_type_extension`.
```
Comment (HeahDude) [inline bundles/override.rst]: 👍

**Review by HeahDude — APPROVED** [review]

File: bundles/inheritance.rst [hunk 2]
```diff
@@ -93,8 +93,8 @@ The same goes for routing files and some other resources.
 
     The overriding of resources only works when you refer to resources with
     the ``@FOSUserBundle/Resources/config/routing/security.xml`` method.
-    If you refer to resources without using the ``@BundleName`` shortcut, they
-    can't be overridden in this way.
+    You need to use the ``@BundleName`` shortcut when refering to resources
+    so they can be successfully overridden.
```
Comment (xabbuh) [inline bundles/inheritance.rst]: There has been confusion in the past about which resources we refer to here. I think we should make more clear that this note does not apply when we talk about templates as the overriding of them works no matter if you use the bundle syntax or the native Twig namespace syntax (I know that the syntax is not exactly the same as what we use to refer to other resources, but that's not so obvious).

File: bundles/override.rst [hunk 3]
```diff
@@ -72,17 +72,8 @@ associations. Learn more about this feature and its limitations in
...
-service as explained in `Services & Configuration`_. This, of course, will only
-work if the type is referred to by its alias rather than being instantiated,
-e.g.::
-
-    $builder->add('name', 'custom_type');
-
-rather than::
-
-    $builder->add('name', new CustomType());
+Existing form types can be modified defining "form type extensions".
```
Comment (wouterj) [inline bundles/override.rst]: I propose: `"form type extensions"` -> `*form type extensions*`

**Review by wouterj — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request merged**